rush.json

```diff
@@ -440,10 +428,6 @@
             "packageName": "@energyweb/origin-backend-core",
             "projectFolder": "packages/origin-backend-core"
         },
-        {
-            "packageName": "@energyweb/migrations",
-            "projectFolder": "packages/tools/migrations"
-        },
         {
             "packageName": "@energyweb/migrations-irec",
             "projectFolder": "packages/tools/migrations-irec"
```

Comment:
What's the reason for removing them from thee project index?

Reply:
So that we won't build them or run any other commands in them

Reply:
Hmm, I guess that's fine for now until we move them to a separate repo.